cli_tools_tests/infra/README

```diff
@@ -0,0 +1,11 @@
+This directory contains terraform definitions that are used with module and e2e testing.
```

Comment:
Add some background on Terraform for those who are not aware of what Terraform is.

Also, definitions for what? GCP projects?

Reply:
punting on this since I removed the Terraform configs in this PR